strongbox-testing/strongbox-testing-core/src/main/java/org/carlspring/strongbox/util/TestFileUtils.java

```diff
@@ -21,4 +30,50 @@ public static void deleteIfExists(File file)
         }
     }
 
+    public static void generateFile(ZipOutputStream zos,
+                                    long size)
+            throws IOException
+    {
+        generateFile(zos, size, "file-with-given-size");
+    }
+
+    public static void generateFile(ZipOutputStream zos,
+                                    long size,
+                                    String name)
+            throws IOException
+    {
+        ZipEntry ze = new ZipEntry(name);
+        zos.putNextEntry(ze);
+
+        RandomInputStream ris = new RandomInputStream(size);
+
+        byte[] buffer = new byte[4096];
+        int len;
+        while ((len = ris.read(buffer)) > 0)
+        {
+            zos.write(buffer, 0, len);
+        }
+
+        ris.close();
+        zos.closeEntry();
+    }
+
+    public static void generateFile(OutputStream bos,
+                                    long size)
+            throws IOException
+    {
+        bos.write("data = \"".getBytes(StandardCharsets.UTF_8));
+
+        OutputStream dataOut = Base64.getEncoder().wrap(bos);
+        RandomInputStream ris = new RandomInputStream(size);
```

Comment:
Same as above 
Use [The try-with-resources Statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) , please.